.github/workflows/ci-baseline.yml

```diff
@@ -82,45 +52,23 @@ jobs:
           #- { os: ubuntu-latest, image: intel_oneapi-eigen_3.3.7-gtest, mode: Release }
           #- { os: ubuntu-latest, image: intel_oneapi-eigen_3.3.7-gtest, mode: Debug }
 
-    # https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idruns-on
-    # The type of machine to run the job on
     runs-on: ${{ matrix.config.os }}
-
-    # https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idcontainer
-    # A container to run any steps in a job that don't already specify a container.
     container: pressiomodelreduction/${{ matrix.config.image }}
-
-    # https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idenv
-    # A map of environment variables that are available to all steps in the job
     env:
       num_cpus: 2
-      pressio_src: /__w/pressio/pressio
-      pressio_build: /out/pressio/build
       eigen_inc_dir: /usr/local/eigen/install/include/eigen3
       gtest_dir: /usr/local/gtest/install
       small: small
       medium: medium
       large: large
 
-    ############################## STEPS in `CI` job ##############################
-    # https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idsteps
-    # A job contains a sequence of tasks called steps.
     steps:
+      - uses: actions/checkout@v3
```

Comment:
what is difference between v3 and v2?

Reply:
v3 is the latest (stable) version. v2 is deprecated and causes GitHub to spit out following warnings:
```
The following actions uses node12 which is deprecated and will be forced to run on node16: actions/checkout@v2. For more info: https://github.blog/changelog/2023-06-13-github-actions-all-actions-will-run-on-node16-instead-of-node12-by-default/
```

From the functionality point of view, there should be no difference (for this workflow).